portfolio/src/main/webapp/script.js

```diff
@@ -1,17 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Probably shouldn't remove license lines from files, generally speaking.

Reply:
Thanks for the feedback!